.github/workflows/maestro_check-schemas.yaml

```diff
@@ -44,12 +43,9 @@ jobs:
         if: steps.filter.outputs.demos == 'true'
         run: |
           # Go to project root (from maestro folder) and run the update script
-          cd maestro
-          cd ..
           chmod +x tools/update_all_readme.sh
           ./tools/update_all_readme.sh
```

Comment:
looks like this script got renamed somehow... seems like it should be update_readme.sh instead (drop the _all_)

Reply:
yeah saw that. How it was working before?? anyhow. Fixed